idd/Energy+.idd.in

```diff
@@ -52999,9 +53001,13 @@ AirLoopHVAC:UnitarySystem,
        \type choice
        \key Load
        \key SetPoint
+       \key ASHRAE90VariableFan
```

Comment:
It was discussed early on, but if it strikes you as odd now, then we should revisit it.